functests/xml-tests.yaml

```
@@ -87,6 +134,33 @@ tests:
       </unit>
     normalize: none
 
+  - name: "Balanced Ternary"
```

This appears to be a duplicate test name, which could cause confusion if it was to fail. Could it be renamed?
```suggestion
  - name: "Negative Balanced Ternary"
```